.github/workflows/test_linux_jax_wheels.yml

```diff
@@ -0,0 +1,219 @@
+        required: false
+        type: string
+        default: "rocm-jaxlib-v0.7.1"
+      test_runs_on:
+        description: Runner label to use for testing (GPU-capable recommended)
+        required: false
+        type: string
+
+  workflow_dispatch:
+    inputs:
+      amdgpu_family:
+        type: choice
+        options:
+          - gfx110X-dgpu
+          - gfx1151
+          - gfx120X-all
+          - gfx94X-dcgpu
+          - gfx950-dcgpu
+        default: gfx94X-dcgpu
+      release_type:
+        type: choice
+        options: [dev, nightly]
+        default: dev
+      s3_subdir:
+        type: string
+        default: v2-staging
+      rocm_version:
+        type: string
+        required: true
+      tar_url:
+        type: string
+        required: true
+      python_versions:
+        type: string
+        required: true
+        default: "3.12"
+      jax_ref:
+        type: string
+        default: master
+      jax_test_branch:
+        type: string
+        description: JAX test repo branch to get unit tests from
+        default: "rocm-jaxlib-v0.7.1"
+      test_runs_on:
+        type: string
+        description: Runner label to use for testing (GPU-capable recommended)
+        default: "linux-mi325-1gpu-ossci-rocm"
+
+permissions:
+  id-token: write
+  contents: read
+  packages: write
+
+jobs:
+  build_test_push:
+    name: Build, Test, then Push JAX Docker | ${{ inputs.amdgpu_family }}
+    runs-on: ${{ github.repository_owner == 'ROCm' && 'azure-linux-scale-rocm' || 'ubuntu-24.04' }}
+    env:
+      S3_BUCKET_PY: therock-${{ inputs.release_type }}-python
+      WHEELHOUSE_DIR: ${{ github.workspace }}/jax/wheelhouse
+    steps:
+      - name: Checkout TheRock
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+
+      - name: Checkout rocm-jax (plugin + build scripts)
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          path: jax
+          repository: rocm/rocm-jax
+          ref: ${{ inputs.jax_ref }}
+
+      - name: Setup Python
+        uses: actions/setup-python@e797f83bcb11b83ae66e0230d6156d7c80228e7c # v6.0.0
+        with:
+          python-version: "3.12"
+
+      - name: Install AWS CLI
+        run: bash ./dockerfiles/install_awscli.sh
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@7474bc4690e29a8392af63c5b98e7449536d5c3a # v4.3.1
+        with:
+          aws-region: us-east-2
+          role-to-assume: arn:aws:iam::692859939525:role/therock-${{ inputs.release_type }}-releases
+
+      - name: Prepare wheelhouse and download JAX wheels
+        shell: bash
+        working-directory: jax
+        run: |
+          set -euo pipefail
+
+          mkdir -p "${WHEELHOUSE_DIR}"
+
+          PY_VER="${{ inputs.python_versions }}"
+          CP="${PY_VER//./}"
+          BASE="s3://${{ env.S3_BUCKET_PY }}/${{ inputs.s3_subdir }}/${{ inputs.amdgpu_family }}/"
```

Comment:
`BASE` could be renamed to something more descriptive, making it clear that it copies from a remote bucket.